llvm/lib/Transforms/Utils/BuildLibCalls.cpp

```diff
@@ -66,6 +67,14 @@ static bool setIsCold(Function &F) {
   return true;
 }
 
+static bool setNoReturn(Function &F) {
```

Comment:
`noreturn` has been added by frontend. These libfuncs are marked by `[[noreturn]]` or `_Noreturn`.

Reply:
My understanding of TLI is it should stand alone if say we are linking w.o headers (not that its the most common thing).

I don't have particularly strong preference here, just thought I'd add it when I was doing terminate/__cxa_throw.